image-index.md

```diff
@@ -55,19 +55,29 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the operating system version.
+        Image indexes SHOULD use, and implementations SHOULD understand the following convention:
+        - `windows`: `sprintf("%d.%d.%d", dwMajorVersion, dwMinorVersion, dwBuildNumber)` with the fields of the `OSVERSIONINFOEX` structure returned by `GetVersionEx()`. e.g., `10.0.10586`.
+        If an operating system is not listed here, it SHOULD be submitted to this specification for standardization.
 
     - **`os.features`** *array of strings*
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory OS feature (for example on Windows `win32k`).
+        A convention for other features SHOULD be submitted to this specification for standardization.
```

Comment:
On Linux, this could include an array of `CONFIG_` entries needed in the host kernel.  For example, `["FUSE_FS"]` or `["NFSD_V4"]`.  Or it could be implementation-defined on Linux until we want to get around to defining some values.